### DISCUSSION TITLE
why this is not build with rust ?

### DISCUSSION THREAD

**Discussion by moolight-seashell** [tianocore/edk2#5455] [General]
probably a stupid question here but I was wondering why

and wouldn't rust security be of interest on such a fundamental low level code?

**Comment by kevindavisinsyde** [comment]
Mainly because UEFI predates Rust. There have been a couple of attempts to prototype some UEFI support in Rust. There was a comment a few weeks ago about Microsoft upstreaming their Rust UEFI platform. I don’t know anything about the status. Since TianoCore is an open source project you are welcome to work on it.TianoCore is used by hundreds of projects around the world by dozens of manufacturers. It’s so critical to the success of these projects that its stability is very important. Thanks,Kevin_______________________________________________________________On Mar 9, 2024, at 3:25 PM, moolight-seashell ***@***.***> wrote:﻿
probably a stupid question here but I was wondering why
and wouldn't rust security be of interest on such a fundamental low level code?
